src/components/KFileUpload/KFileUpload.vue

```diff
@@ -214,7 +215,7 @@ const onFileChange = (evt: any): void => {
 
   const fileSize = fileInput?.value[0]?.size
 
-  hasUploadError.value = fileSize > maximumFileSize.value
+  hasUploadError.value = fileSize as Number > maximumFileSize.value
```

Comment:
suggestion
```suggestion
  hasUploadError.value = Number(fileSize) > maximumFileSize.value
```

Reply:
This doesn't solve the issue, still need to do `as Number` (the error is `Operator '>' cannot be applied to types 'number' and 'Number'`)